go/exoservice/exoservice_test.go

```diff
@@ -72,9 +72,10 @@ func FeatureContext(s *godog.Suite) {
 
 	s.Step(`^receiving a "([^"]*)" message(?: with (?:id "([^"]*)")?(?:(?: and )?sessionId "([^"]*)")?)?$`, func(name, id, sessionId string) error {
 		message := structs.Message{
-			ID:        id,
-			Name:      name,
-			SessionID: sessionId,
+			ActivityID: id,
+			ID:         id,
```

Comment:
id isn't as important anymore and we should use different id and activity id. Please update the step to accept an activityId instead of an id and generate a random message id.